src/components/editor/Status/Invalid.tsx

```diff
@@ -0,0 +1,17 @@
+import CloudOffIcon from '@mui/icons-material/CloudOff';
+import { FormattedMessage } from 'react-intl';
+
+interface Props {
+    iconSize: number;
+}
+
+function Invalid({ iconSize }: Props) {
+    return (
+        <>
+            <FormattedMessage id="common.invalid" />
+            <CloudOffIcon sx={{ fontSize: iconSize }} />
+        </>
+    );
+}
```

Comment:
New display for invalid. hoping the "Disabled cloud" I con will communicate that things were not saved